strapi/gcp-deployment/package.json

```diff
@@ -8,7 +8,10 @@
     "start": "strapi start",
     "build": "strapi build",
     "strapi": "strapi",
-    "gcp-build": "strapi build"
+    "gcp-build": "strapi build",
+    "app-deploy": "gcloud app deploy app.yaml --project=ch-test-strapi --bucket=gs://ch-test-strapi-app-bucket",
```

Comment:
Note: Hardcoding the flags is fine for this experiment, but for production, let's use env vars. That will make it easier to do things like switch between staging and production.

e.g. we could have
```
strapi/
├── staging/
│   └── .env
└── production/
    └── .env
```

So working with `staging/` would be something like:

```
cd strapi/staging;
source .env;
gcloud app deploy app.yaml --project=$GCLOUD_PROJECT --bucket=$GAE_BUCKET;
```
direnv might make sense for this use case.